### PR TITLE
Adds vysokan/himean clothing to loadout

### PR DESCRIPTION
**Pull request by kyres1**
Title. Adds three vysokan things and two himean things to the loadout. Requested by Schwann.

There's a feminine and a masculine version of the chokha, which is a coat.  There is also a fur cap.

For himeo, there's a fishfur coat and an edgy hat.

This was originally supposed to include Elyran holoclothes but I got lazy so I reverted it. I think it worked???

image

### CODE REVIEW

**Comment by kyres1** [comment]
!review

File: code/modules/clothing/factions/vysoka.dm [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	icon = 'icons/clothing/suits/coats/vysoka_chokha_female.dmi'
+	icon_state = "chokha_vysoka_f"
+	item_state = "chokha_vysoka_f"
+	contained_sprite = TRUE
+
+/obj/item/clothing/head/soft/vysoka
+	icon_state = "cap_vysoka"
+	item_state = "cap_vysoka"
+	icon = 'icons/clothing/head/vysoka_cap.dmi'
+	name = "vysokan fur cap"
```
Comment (Geevies) [inline code/modules/clothing/factions/vysoka.dm:23]: name
desc
icon
the rest

use the same format as the others bleez

Comment (kyres1) [reply code/modules/clothing/factions/vysoka.dm:23]: OK boss will do soon

File: code/modules/client/preference_setup/loadout/loadout_suit.dm [hunk 2]
```diff
@@ -324,3 +324,19 @@
 	cardigan["sweater cardigan"] = /obj/item/clothing/suit/storage/toggle/cardigan/sweater
 	cardigan["argyle cardigan"] = /obj/item/clothing/suit/storage/toggle/cardigan/argyle
 	gear_tweaks += new/datum/gear_tweak/path(cardigan)
+
+/datum/gear/suit/himeo
```
Comment (Alberyk) [inline code/modules/client/preference_setup/loadout/loadout_suit.dm:328]: This needs the GEAR_HAS_DESC_SELECTION flag.

File: code/modules/client/preference_setup/loadout/loadout_suit.dm [hunk 3]
```diff
@@ -324,3 +324,19 @@
...
+	display_name = "himean coat"
+	path = /obj/item/clothing/suit/storage/toggle/himeo
+
+/datum/gear/suit/vysoka
```
Comment (Alberyk) [inline code/modules/client/preference_setup/loadout/loadout_suit.dm:332]: This needs the GEAR_HAS_DESC_SELECTION flag.

**Pull request merged**